src/Components/Form/questions.js

```diff
@@ -0,0 +1,68 @@
+export const questionSet = [
+    {
+        question: 'Does your group include kids?', 
+        choices: ['yes', 'no'],
+        activities: [],
+        answerType: 'default'
+    }, 
+    {
+        question: 'Which activities excite you right now?',
+        choices: ['movies', 'board games', 'music', 'card games', 'podcasts'],
+        activities: [],
+        answerType: 'default'
+    },
+    {
+      question: 'How old is too old?',
+      choices: ['2010\'s', '2000\'s','1990\'s', '1980\'s'],
+      activities: ['music'],
```

Comment:
Now that we've separated our questions based on a single activity, we should change `activities` in each of these objects to `activity` with a string data type for the value.

Reply:
Agree 💯

Reply:
Yeah that makes sense!